doc/active_docs/billing_api.json

```diff
@@ -232,13 +232,6 @@
                 "required": [
                   "access_token"
                 ]
-              },
```

Comment:
LOL, where did this come from? :sweat_smile:  Doesn't seem to have anything to do with this API :grimacing: